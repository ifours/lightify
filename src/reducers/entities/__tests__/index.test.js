import reducer from '../index';

describe('Entities root reducer', () => {
  it('should return backbone of stities state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot();
  });
});
