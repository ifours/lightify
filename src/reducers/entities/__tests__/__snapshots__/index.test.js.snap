// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Entities root reducer should return backbone of stities state 1`] = `
Object {
  "albums": Object {},
  "playlists": Object {},
  "tracks": Object {},
}
`;
